.flake8

```diff
@@ -0,0 +1,18 @@
+[flake8]
```

Comment:
I'd keep it out of the scope; it's a bit opinionated to include linters, etc. 

Even requirements.in could be an overkill here, though no strong opinion on that. The issue is that unless people get used to .in/.txt tooling, they'd be getting old packages.